templates/cisco_ios_show_license.template

```diff
@@ -0,0 +1,16 @@
+Value FEATURE (\S+)
+Value PERIOD_LEFT (.*)
+Value PERIOD_USED (.*)
+Value LICENSE_TYPE (.*)
+Value LICENSE_STATE (.*)
+Value LICENSE_COUNT (.*)
+Value LICENSE_PRIORITY (.*)
+
+Start
+  ^Index\s\d\sFeature:\s${FEATURE}\s+$$
+  ^\s+Period left:\s${PERIOD_LEFT}$$
+  ^\s+Period Used:\s${PERIOD_USED}$$
+  ^\s+License Type:\s${LICENSE_TYPE}$$
+  ^\s+License State:\s${LICENSE_STATE}$$
+  ^\s+License Count:\s${LICENSE_COUNT}$$
+  ^\s+License Priority:\s${LICENSE_PRIORITY}$$ -> Record
```

Comment:
Please update to:
```
  ^\s*$$
  ^. -> Error
```

Also, anywhere in the state that is a space, can you change it to **\s*** or **\s+** depending on the case. It looks like most will be **\s**. Please do the same for all the regular **\s** as well.

Reply:
Hey @FragmentedPacket im sorry for being dense, but I dont understand this part from your comments.
```
  ^\s*$$
  ^. -> Error
```
Can you talk me through this a bit more please ?

Reply:
We require each state (ex. **Start**) to end with those two lines. So you would add those after L16.

What it does is to make sure all lines are being accounted for from the output of the show command and any whitespace. 

Is that what you were looking for?

Reply:
Ah yes, that is perfect thank you. I misunderstood the purpose and positioning, thank you very much for the explanation! I will get back with an update PR.

Reply:
Thanks again for the help @FragmentedPacket I have made the updates now.